plugins/generator-1.15.2/forge-1.15.2/procedures/strike_lightning.java.ftl

```diff
@@ -1 +1 @@
-if(world instanceof ServerWorld) ((ServerWorld)world).addLightningBolt(new LightningBoltEntity(world.getWorld(),(int)${input$x},(int)${input$y},(int)${input$z},false));
\ No newline at end of file
+if(world instanceof ServerWorld) ((ServerWorld)world).addLightningBolt(new LightningBoltEntity(world.getWorld(),(int)${input$x},(int)${input$y},(int)${input$z},${input$isCosmetic}));
```

Comment:
Please add default values to the generator like in this commit when changing already existing blocks to extend behaviour: https://github.com/MCreator/MCreator/commit/f0cbd0b7c1b70220a6fa5f9cfee30b8c37792612 Try to also make sure it doesn't change the behaviour of already existing blocks without this field.

Reply:
Done!